pipelines/_ingredients/ApPipeWithFakes.yaml

```diff
@@ -41,8 +41,6 @@ tasks:
       connections.fakesType: parameters.fakesType
       insertFakes.doSubSelectSources: True
       insertFakes.select_col: 'isVisitSource'
-      calibrate.photoCal.match.referenceSelection.magLimit.fluxField: i_flux
-      calibrate.photoCal.match.referenceSelection.magLimit.maximum: 22.0
```

Comment:
I appreciate that you don't want these in the generic pipeline. Since you are removing them here, they ought to land in each camera's `processVisitFakes` pipeline instead. In practice, both HSC and DECam use refcats that have the `i_flux` flux field, and this is the best proxy config available for "only use sources with a decent signal-to-noise-ratio for photometric calibration."

Reply:
Except that a lot of the confusion surrounding these overrides came from the fact that `processVisitFakes` is _not_ supposed to run `photoCal`, and ignores these fields -- which is why none of the `processCcdWithFakes.py` files bothered setting them. So I really think the best solution is to not mention them in the pipelines at all, and trust that the config files are already doing the (generically) right thing.

Reply:
Now that I appreciate this nuance, I retract my request to leave these configs anywhere in "calibrateTask Take 2" that happens inside processCcdWithFakes.